dcpy/data/compare.py

```diff
@@ -0,0 +1,320 @@
+import pandas as pd
+from pydantic import BaseModel
+from typing import TypeVar, Generic
+
+from dcpy.models.base import IndentedPrint
+from dcpy.utils import postgres
+
+
+T = TypeVar("T")
+
+
+class SimpleComparison(IndentedPrint, Generic[T]):
+    left: T
+    right: T
+
+
+class EntryWiseComparison(BaseModel, Generic[T]):
```

Comment:
mind throwing a docstring on these classes? I should probably already know what EntryWise means in this context, but alas...

Reply:
Just dropped it entirely - wasn't the most aptly named, and it got superseceded by the columns comparison class in my use case